README.md

```diff
@@ -1,4 +1,38 @@
-# pgSTAC CDK construct
+# eoAPI CDK Construct
+
+eoapi-cdk is a package of [AWS CDK constructs](https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/constructs-best-practices.html) designed to encapsulate eoAPI runtime code and best practices as simple reusable components.

```

Comment:
```suggestion
eoapi-cdk is a package of [AWS CDK constructs](https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/constructs-best-practices.html) designed to encapsulate eoAPI services and best practices as simple reusable components.

```

Reply:
If I am understanding your comment correctly vincent, this relates to https://github.com/developmentseed/cdk-pgstac/issues/4 (we currently have the stac ingestor runtime stored here, whereas it ideally should be maintained separately)